catalog/core/catalog-core-standardframework/src/main/java/ddf/catalog/impl/CatalogFrameworkImpl.java

```diff
@@ -188,6 +188,8 @@
 
     private static final String FANOUT_MESSAGE = "Fanout proxy does not support create, update, and delete operations";
 
+    public static final String METACARD_PROPERTY_NAME = "metacard";
```

Comment:
I wouldn't include this constant here since that will force anyone wanting to use to depend on `CatalogFrameworkImpl`. There is a `Constants` class in `catalog-core-api`.